csrc/scheduler/hopper_multi_matmul.cpp

```diff
@@ -1019,8 +1019,20 @@ void HopperMultipleMatmulScheduler::scheduleEpilogue() {
     // TODO: Use stmatrix to load from registers to shared memory
     constexpr int64_t stmatrix_tile_m = 16;
     constexpr int64_t stmatrix_tile_n = 16;
-    constexpr int64_t tma_m = 16;
-    constexpr int64_t tma_n = 64;
+    const int64_t tma_m = getM(params_->mma_macro);
```

Comment:
A comment here would be appreciated. We should indicate that the TMA size is linked to the macro size because that tile size determines which threads produce which elements of the output.

Reply:
Is this the instruction tile size? Or the warp(group) tile size.. :-D

Reply:
At the moment, it is the warp group tile size. It could be the cta tile size but this didn't work correctly. I'm not sure if storing the cta tile is ideal because it would use more shared memory.

Reply:
That's what I was thinking. So I think we should use `params_->tile_sizes.warp_tile.m` instead of `getM(params_->mma_macro)` here.

Reply:
>  I'm not sure if storing the cta tile is ideal because it would use more shared memory.

We have typically stored the whole CTA tile for unswizzling on Ampere...

Reply:
On Ampere, the `wmma` creates a (16, 16) tile, so you can accumulate a larger tile like (64, 64).
On Hopper, the `wgmma` creates a (64, N) tile, so it is already quite large. There isn't much benefit for accumulating a large smem tile for tma store. You'd like to balance the shared memory for the operand pipelines and occupancy.